src/__tests__/calculate.test.jsx

```diff
@@ -0,0 +1,38 @@
+import calculate from '../logic/calculate';
+
+describe('Check calculate function behavior according to the operation', () => {
+  test('Sum operation button', () => {
```

Comment:
I have gone through all of your tests and i noticed that you are not still using the snapshot and i strongly believe that it is because you arent testing the how something is rendered in the DOM. To get this right you should be using the jest method called `tomatchSnaphot`. Kindly do so to stay in sync with the requirement.  Check out my [repo](https://github.com/DuaneDave/Math-magicians/tree/dev/src/test) to have a lead on what is expected.